packages/client/components/ActionMeetingSidebar.tsx

```diff
@@ -57,30 +59,31 @@ const ActionMeetingSidebar = (props: Props) => {
             const phaseCount =
               phaseType === 'agendaitems' && agendaItems ? agendaItems.length : undefined
             return (
-              <NewMeetingSidebarPhaseListItem
-                handleClick={canNavigate ? handleClick : undefined}
-                isActive={
-                  phaseType === 'agendaitems'
-                    ? localPhaseType !== '' && blackList.includes(localPhaseType)
-                    : localPhaseType === phaseType
-                }
-                isCollapsible={collapsiblePhases.includes(phaseType)}
-                isFacilitatorPhase={phaseType === facilitatorPhaseType}
-                isUnsyncedFacilitatorPhase={
-                  isUnsyncedFacilitatorPhase && phaseType === facilitatorPhaseType
-                }
-                isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
-                key={phaseType}
-                phaseCount={phaseCount}
-                phaseType={phaseType}
-              >
+              <Fragment key={phaseType}>
```

Comment:
The header is now a peer of the list items.
This is good because the DOM tree is a little more shallow, but also it means we can set minHeight on the list item instead of having to set it on the parent, which means fewer props to pass down